### ISSUE TITLE
[BUG] Daily cronjob of Spamassassin KAM is failing

### ISSUE THREAD

**Issue by wolkenschieber** [docker-mailserver/docker-mailserver#2458]
### Miscellaneous first checks

- [X] I checked that all ports are open and not blocked by my ISP / hosting provider.
- [X] I know that SSL errors are likely the result of a wrong setup on the user side and not caused by DMS itself. I'm confident my setup is correct.

### Affected Component(s)

Spamassassin KAM

### What happened and when does this occur?

The daily cronjob of Spamassassin KAM has failed and mailed this message:

> **Cron <root@mail> test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.daily )**
> /etc/cron.daily/spamassassin_kam:
> Reloading SpamAssassin Mail Filter Daemon: No spamd found running; none killed.
> run-parts: /etc/cron.daily/spamassassin_kam exited with return code 1

### What did you expect to happen?

```Markdown
No report from cronjob.
```

### How do we replicate the issue?

I've set those environment variables in the compose file:
```Markdown
    environment:
      - SSL_TYPE=letsencrypt
      - SSL_DOMAIN=mail.domain.tld
      - ENABLE_CLAMAV=1
      - ENABLE_DNSBL=1
      - ENABLE_FAIL2BAN=1
      - ENABLE_SPAMASSASSIN=1
      - ENABLE_SPAMASSASSIN_KAM=1
      - ENABLE_POSTGREY=1
      - ENABLE_QUOTAS=1
      - POSTFIX_MESSAGE_SIZE_LIMIT=10240000 # Set the message size limit for all users. ~10 MB
```

### DMS version

v10.5.0

### What operating system is DMS running on?

Linux

### What instruction set architecture is DMS running on?

x86_64 / AMD64

### What container orchestration tool are you using?

Docker Compose

### Other relevant information

Issue #2414 is reporting `spamd` not running in certain configurations.

### What level of experience do you have with Docker and mail servers?

- [ ] I am inexperienced with docker
- [ ] I am inexperienced with mail servers
- [ ] I am uncomfortable with the CLI

### Code of conduct

- [X] I have read this project's [Code of Conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md) and I agree
- [X] I have read the [README](https://github.com/docker-mailserver/docker-mailserver/blob/master/README.md) and the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/) and I searched the [issue tracker](https://github.com/docker-mailserver/docker-mailserver/issues?q=is%3Aissue) but could not find a solution

**Comment by georglauterbach** [comment]
ref #2456

**Comment by Dubbeldrank** [comment]
I can confirm this.

**Comment by georglauterbach** [comment]
It is added, but the second part of the `&&` produces the error. Sadly, we do not provide backports for this... I'm sorry for the inconvenience.

**Comment by casperklein** [comment]
The fix is available in the current `:edge` build and later in the upcoming version 11.

**Comment by ghnp5** [comment]
Ah... I was going to report this :)

```
/etc/cron.daily/aaa_spamassassin_KAM:
Update available for channel kam.sa-channels.mcgrail.com: 1646438287 -> 1646497491
http: (curl) GET http://sa-update-kam.snb.it/1646497491.tar.gz, success
http: (curl) GET http://sa-update-kam.snb.it/1646497491.tar.gz.sha512, success
Update was available, and was downloaded and installed successfully
run-parts: /etc/cron.daily/spamassassin_kam exited with return code 1
```

SpamAssassin runs under Amavis, so "reload" doesn't work.

`/etc/cron.daily/spamassassin` runs "reload", but the script itself doesn't seem to report as returning exit code 1.

**Comment by woeisme** [comment]
Sorry to reopen a dead thread but how to do we fix this?

**Comment by georglauterbach** [comment]
> Sorry to reopen a dead thread but how to do we fix this?

https://github.com/docker-mailserver/docker-mailserver/issues/2458#issuecomment-1059744339

This should be fixed in `v11.0.0` or `v11.1.0`. SA jobs may fail from time to time because there are problems on the remote server. If you see the error for a prolonged period of time, please feel free to open a new issue and describe your problem.

**Comment by ghnp5** [comment]
I confirm this fixed since the last version 👍
